cmake/test.cmake

```diff
@@ -197,11 +197,22 @@ finally:
         list(APPEND mayaUsd_varname_PATH $ENV{PYTHONHOME})
     endif()
 
-    # inherit PATH and PYTHONPATH from ENV to get USD entries
-    # these should come last (esp PYTHONPATH, in case another module is overriding
+    # Adjust PATH and PYTHONPATH to include USD.
+    # These should come last (esp PYTHONPATH, in case another module is overriding
     # with pkgutil)
-    list(APPEND mayaUsd_varname_PATH $ENV{PATH})
-    list(APPEND mayaUsd_varname_PYTHONPATH $ENV{PYTHONPATH})
+   if (DEFINED MAYAUSD_TO_USD_RELATIVE_PATH)
+        set(USD_INSTALL_LOCATION "${CMAKE_INSTALL_PREFIX}/${MAYAUSD_TO_USD_RELATIVE_PATH}")
+    else()
+        set(USD_INSTALL_LOCATION ${PXR_USD_LOCATION})
+    endif()
+    list(APPEND mayaUsd_varname_PYTHONPATH
+        "${USD_INSTALL_LOCATION}/lib/python")
+    if(IS_WINDOWS)
+        list(APPEND mayaUsd_varname_PATH
+            "${USD_INSTALL_LOCATION}/bin")
+        list(APPEND mayaUsd_varname_PATH
+            "${USD_INSTALL_LOCATION}/lib")
+    endif()
```

Comment:
I reverted my change in usd.cmake where I was using the USD from install folder because it was wrong. In our internal builds USD is only copied to install folder during install stage. During configure stage we'll need to use USD from the given input USD location.

However at test time we now check if we were given a USD relative path. If so we use'll that one and add USD to the PYTHONPATH from there. 

Here I've only add USD to PATH when on Windows, just like usd.cmake does.

I believe that even all the PATH items above should only need to be done on Windows, but I didn't change them so as to not pollute this PR.